common-data/config.ts

```diff
@@ -38,7 +38,7 @@ const config: CommonDataConfig = {
       jsonFilename: "lease-choices.json",
       typeName: "LeaseChoice",
       exportLabels: true,
-      filterOut: ["NOT_SURE"],
+      filterOut: ["NO_LEASE"],
```

Comment:
Hmm, since the front-end _does_ actually need the label for this in `frontend/lib/account-settings/nyc-address-settings.tsx`, in order to support users who chose the option in the past, maybe this list should be empty, and instead we filter out this option when we show the radio button choices to the user in the front-end?  (And perhaps even include the option if the user already has it set, so they're not confused about its disappearance?)

If that doesn't make sense I could try issuing a PR to this PR to show what I mean.

Reply:
I made this as a PR in #2155, feel free to merge that into this PR if you want.